include/exadg/time_integration/time_int_bdf_base.h

```diff
@@ -220,12 +220,6 @@ class TimeIntBDFBase : public TimeIntBase
   virtual void
   prepare_vectors_for_next_timestep() = 0;
 
-  /*
-   * Solve the current time step.
-   */
-  virtual void
-  solve_timestep() = 0;
-
```

Comment:
This looks like a change but the method is already in the base class, as indicated by the compiler warning I got.

Reply:
ah, makes sense.